packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -238,7 +263,17 @@ class VirtualServiceBackend extends Backend {
       virtualServiceBackend: {
         virtualService: {
           virtualServiceName: this.virtualService.virtualServiceName,
-          clientPolicy: this.clientPolicy?.bind(_scope).clientPolicy,
+          clientPolicy: this.tlsClientPolicy !== undefined ?
+            {
+              tls: {
+                ports: this.tlsClientPolicy.ports,
+                enforce: this.tlsClientPolicy.enforce,
+                validation: {
+                  trust: this.tlsClientPolicy.tlsValidationContext.trust.bind(_scope).virtualNodeClientTlsValidationContextTrust,
```

Comment:
Now that it's being used, `_scope` should be renamed to `scope`.

Reply:
Addressed.